packages/nc-gui/components/account/License.vue

```diff
@@ -1,12 +1,14 @@
 <script lang="ts" setup>
 import { useNuxtApp } from '#app'
 import { message } from 'ant-design-vue'
-import { extractSdkResponseErrorMsg, useApi } from '#imports'
+import { extractSdkResponseErrorMsg, useApi,useGlobal } from '#imports'
 
 const { api, isLoading } = useApi()
 
 const {$e} = useNuxtApp()
 
+const {loadAppInfo} = useGlobal()
```

Comment:
Spacing issue? Formater not working?

Reply:
Somehow linting script not changed anything, so I did it manually and pushed.